tests/Behat/WebtoolsCookieConsentContext.php

```diff
@@ -39,30 +38,30 @@ public function gatherContexts(BeforeScenarioScope $scope) {
   }
 
   /**
-   * Enables the Media module.
+   * Enables the Media and the Path modules.
    *
    * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
    *   The scope.
    *
    * @beforeScenario @remote-video
    */
   public function enableModule(BeforeScenarioScope $scope): void {
-    \Drupal::service('module_installer')->install(['oe_media']);
+    \Drupal::service('module_installer')->install(['oe_media', 'path']);
```

Comment:
Why isn't this also using `WebtoolsExtentionsContext` logic to enable and disable modules?
It would go in the direction of the requested change in https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1999 .

Reply:
Using for enabling/disabling of this modules `WebtoolsExtentionsContext` I have found for now is problematic by follow reasons:

1. As far as I remember this tag ('@remote-video') used not only in oe_webtools but also in behat tests of other components (including in ewcms). So replacing probably it should be done in more global level (applying approach for all other components), of cause if coordination team agree with that.
2. Also I have found that oe_media still have problem with reinstalling (have error regarding already imported configs).

Sorry for long explanation, most simple reason of this choice that by `WebtoolsExtentionsContext` we could enable modules specifically for Behat features (not scenarios), because we have problem with order of Before/AfterScenario method executing.

Reply:
Makes sense and you're right, OPENEUROPA-1999 is a better place to address this in a global level.